ticketing/packages/uiPlugin/README.md

```diff
@@ -0,0 +1,197 @@
+# VCD UI Plugin Seed #
```

Comment:
I think this file should be removed or dramatically simplified, as it talks about different project.